.hyperlint/styles/config/vocabularies/hyperlint/accept.txt

```diff
@@ -0,0 +1,12 @@
+validator
```

Comment:
This is your custom dictionary: [Read more here.](https://docs.hyperlint.com/ai-reviewer/custom-style-guide)

Reply:
Great! I'll expand on this when I do some docs updates tomorrow.